src/Sinch/Core/Http.cs

```diff
@@ -79,8 +79,85 @@ internal class Http : IHttp
             return Send<object, TResponse>(uri, httpMethod, null, cancellationToken);
         }
 
+
+        public async Task<TResponse> Send<TResponse>(Uri uri, HttpMethod httpMethod, HttpContent httpContent,
```

Comment:
its not same, this takes http content and respects the given content (we have multipart/form-data for attaching files to request). the other takes an empty body. maybe it should be refactored

Reply:
Ah, yes. To not duplicate functionality I see how can we refactor that, for sure method `SendMultipart` should be added to `Http` and utilize for generic `Send` which takes `HttpContent` instead of generic param

Reply:
not sure i follow here